submissions/OwoeyeGbenga/landing_page/css/style.css

```diff
@@ -0,0 +1,389 @@
+html {
+  box-sizing: border-box;
+}
+*, *:before, *:after {
+  box-sizing: inherit;
+}
+body {
+	font-family: "Cabin", Arial, Geneva, sans-serif; 
+  background-image:linear-gradient(rgba(0, 0, 0, 0.6), rgba(63, 63, 149, 0.7)), url(../img/fam_cart.jpg);
```

Comment:
Indentation looks good for the most part of this file  👍
Please fix this line and other ones not properly done so it looks 💯

Reply:
Done